src/main/kotlin/com/home/Application.kt

```diff
@@ -23,7 +24,12 @@ class Application(val controller: Controller,
         //TODO: make the port configurable
         port(4567)
 
-        before(Filter { _, response: Response -> response.type("application/json") })
+        before(Filter { _, response: Response ->
+            response.type("application/json")
+            response.header("Access-Control-Allow-Origin", "*")
```

Comment:
Sure, I'd probably do it in scope of issue #20 
We still need to improve the way we configure our application. Is it ok with you if I just adjust the TODO on line 24?

Reply:
That is ok to keep TODO here, but with using Allow All ("*") we are opening API to all another pages.
To avoid security problems and to do not forget fix this issue could we use concrete hardcoded value? e.g. "localhost:3000"